samples/eShopLite/AppHost/mymanifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "resources": {
+    "frontend": {
+      "type": "project.v0",
+      "path": "..\\MyFrontend\\MyFrontend.csproj",
+      "env": {
+        "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES": "true",
+        "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES": "true",
+        "services__petstore": "https://petstore.swagger.io/"
```

Comment:
I'm in favor of having the http service as a resource itself and I wanted to share my thoughts. If I understand correctly, we have to use manifest to do our own deployment. Having a specific resource for Http Service gives us the benefit to identify the external http services in order to allow firewall for those addresses.

Also we use different http service address in development and production for external services, and if multiple project reference that http service, I guess it will be easier to change the manifest from the http resource rather than env of multiple projects.

Reply:
What would a deployment tool do this with this resource?

Reply:
To be fair I don't see any scenario that not having the http resource have any limitations in the deploy but having it at the resource gives me a view of all my applications reference and dependencies and I can easily set the http service URL in one place instead of possibly multiple environment variables of my projects.

If I understand the Redis resource correctly, I can argue that the Redis behaves the same when a connection string is presented. In this case I don't want the deploy to setup a new instance of Redis, I want my app to connect to an existing Redis, right?